src/pkg/cli/tail.go

```diff
@@ -379,6 +381,10 @@ func streamLogs(ctx context.Context, provider client.Provider, projectName strin
 }
 
 func logEntryPrintHandler(e *defangv1.LogEntry, options *TailOptions) error {
+	if options.LogCache != nil {
+		options.LogCache.Add(e.Message)
```

Comment:
we know that often times the relevant error can be anywhere in the logs, so should we use "verbose" logic to ignore the likely informational messages?